src/hooks/useAccessibility.ts

```diff
@@ -181,7 +181,29 @@ function getNextFocusElement(
   return sameLevelFocusableMenuElementList[focusIndex];
 }
 
-export default function useAccessibility<T extends HTMLElement>(
+export const refreshElements = (
+  keys: string[],
+  id: string,
+  elements: Set<HTMLElement>,
+  key2element: Map<string, HTMLElement>,
+  element2key: Map<HTMLElement, string>,
+) => {
+  keys.forEach(key => {
+    const element = document.querySelector(
```

Comment:
`document.querySelector` is too heavy for this simple action. Can we only search for elements in containerRef?

Reply:
We cannot do that, because the submenus are not the part of containerRef.current in DOM, they are added to the end of the document
![image](https://github.com/react-component/menu/assets/26119263/7cbe4883-8c7b-468e-8943-48a47d80bcdc)